src/libs/ajax.js

```diff
@@ -241,9 +239,8 @@ const User = signal => ({
   },
 
   acceptTos: async () => {
-    await fetchOk(
-      `${getConfig().tosUrlRoot}/user/response`,
-      _.mergeAll([authOpts(), { signal, method: 'POST' }, jsonBody({ ...tosData, accepted: true })])
+    await fetchSam('api/users/v1/tos/accept',
+      _.mergeAll([authOpts(), { signal, method: 'POST' }, jsonBody('app.terra.bio/#terms-of-service')])
```

Comment:
Should the URL come from the current location, since the app is deployed at multiple URLs?